file_hook.go

```diff
@@ -7,10 +7,23 @@ import (
 	"path/filepath"
 	"strings"
 	"sync"
+	"unicode"
 
 	"github.com/sirupsen/logrus"
 )
 
+func cleanupModuleName(moduleName string) string {
```

Comment:
I know I could've used a regex but this is much faster and we don't have the hassle of compiling the regex in a multi threaded context: https://stackoverflow.com/a/44009184

Reply:
What is the original problem?

Reply:
You could name a module `../../../..` and just write logs whereever you want